app/javascript/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import configureStore from '../redux/configureStore';
+import Greeting from './Greeting';
+
+const store = configureStore();
+class App extends React.Component {
```

Comment:
- [ ] OPTIONAL --- We should prefer functional components to class based components.